packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -80,7 +80,8 @@ typedef ActionListenerCallback = void Function(Action<Intent> action);
 ///  * [ActionDispatcher], a class that takes an [Action] and invokes it, passing
 ///    a given [Intent].
 abstract class Action<T extends Intent> with Diagnosticable {
-  final ObserverList<ActionListenerCallback> _listeners = ObserverList<ActionListenerCallback>();
+  final ObserverList<ActionListenerCallback> _listeners =
```

Comment:
You seem to be running an autoformatter that doesn't necessarily follow the [Flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo).  You should probably revert all of these formatting changes since they're unrelated to this PR.

This particular change is not a big deal, but some below seem to go against the usual style of the Flutter repo.

Reply:
Coming back to this after looking at the whole PR, I definitely think you should undo all of the formatting changes.  It's hard to read this PR there are so many.